frontend/src/common/utils/api/handler.ts

```diff
@@ -1,20 +1,34 @@
 import type { NextApiRequest, NextApiResponse } from "next";
-import readJson from "./readJson";
+import HTTP_STATUS_CODE from "src/common/constants/HTTP_STATUS_CODE";
+import { readJson } from "src/common/utils/api/readJson";
 
-const getHandler = (path: string) => {
-  const data = readJson(path);
+const getHandler = (filePath: string) => {
+  const data = readJson(filePath);
 
-  async function handler(req: NextApiRequest, res: NextApiResponse) {
+  function handler(req: NextApiRequest, res: NextApiResponse) {
     try {
       const entityId = req.query.entityId;
-      if (data[entityId as keyof typeof data])
```

Comment:
keeping code DRY and readable by creating a variable `const response = data[entityId as keyof typeof data];`